.github/workflows/dis-apim-kustomize-realease.yml

```diff
@@ -0,0 +1,86 @@
+name: Build kustomize oci artifact for Dis APIM Operator
+
+env:
+  ARTIFACT_NAME: kustomize/dis-apim-operator
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'services/dis-apim-operator/config/**'
+      - '.github/workflows/dis-apim-kustomize-release.yml'
+    tags:
+      - 'kustomize-dis-apim-*'
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  latest:
+    name: Build latest from main
+    if: github.ref == 'refs/heads/main'
+    defaults:
+      run:
+        working-directory: ./services/dis-apim-operator/config
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup vars
+        id: vars
+        run: |
+          echo "reponame=${GITHUB_REPOSITORY,,}" >> ${GITHUB_OUTPUT}
+      - name: Setup flux
+        uses: fluxcd/flux2/action@3b42b200d376430f0e24d35f1a600447d92da531
+        with:
+          version: latest
+      - name: Build latest artifact
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+          container_registry=ghcr.io/${{ steps.vars.outputs.reponame }}
+          artifact_name=${{ env.ARTIFACT_NAME }}
+          flux push artifact oci://${container_registry}/${artifact_name}:$(git rev-parse --short HEAD) \
```

Comment:
Not sure I follow what is happening here.
How are you planning to integrate with: https://github.com/Altinn/altinn-platform/blob/main/services/dis-apim-operator/Makefile#L131-L135?

Reply:
I take all the kustomization and other files and backage them up in a OCI image that is pushed to ghcr.

Frankly I wasn't planning on implementing it. I plan to deploy this with flux Kustomization CRDs and that resource reads kustomization.yaml and creates the final result by it self. Then we aren't relying on having a extra step in this pipeline

Reply:
@monteiro-renato thoughts?

Reply:
Through kustomization, the make file leverages the same functionallity.
```yaml
apiVersion: kustomize.toolkit.fluxcd.io/v1
kind: Kustomization
metadata:
  name: dis-apim
  namespace: flux-system
spec:
  interval: 30m
  targetNamespace: dis-apim-operator-system
  sourceRef:
    kind: OciRepository
    name: dis-apim
  path: "./default"
  prune: true
  timeout: 10m
  images:
    - name: controller
      newName: actualcr.azurecr.io/dis-apim/controller
      newTag: <version>
  patches:
        - op: add
          path: /spec/template/metadata/labels/azure.workload.identity~1use
          value:
            "true"
#... Add more environment specific patches...
```